### DISCUSSION TITLE
can we use spark 2.4.7 with the job server v0.10.0

### DISCUSSION THREAD

**Discussion by jainarpitspark** [spark-jobserver/spark-jobserver#1352] [General]
I have to use spark job server with the data proc of GCP which has spark version 2.4.7.
For this I am installing the spark job server on one of the GCP VM and then configuring it to use the dataproc master as spark master. 
Before trying this out I just wanted to check if its going to work with a different sub minor version of spark as officially SJS supports spark 2.4.4.

**Comment by radtke-vale** [comment]
Hi @jainarpitspark ,
You are right, officially release 0.10 was tested only with Spark 2.4.4.
We were using the same 0.10 release with Spark 2.4.6 and didn't notice any problems, so most likely it will also work for 2.4.7. 

But you may need to recompile Jobserver project using environment variable `SPARK_VERSION=2.4.7`.

**Reply by venkatkrishna110** [reply]
Hi @vglagoleva ,
can I use release 0.10.2 with spark-2.4.4 in yarn-client mode ?
